ScheduledExecutorServiceBasicTest.scheduleWithRepetition

_maintenance-3.x_ (commit 11febd59d9417d97487f5d93a6185a3c2a812795)

Failed on IbmJDK7: http://jenkins.hazelcast.com/view/Official%20Builds/job/Hazelcast-3.maintenance-IbmJDK7/12/testReport/com.hazelcast.scheduledexecutor/ScheduledExecutorServiceBasicTest/scheduleWithRepetition/

Stacktrace:
```
java.lang.AssertionError: expected:<0> but was:<2>
	at org.junit.Assert.fail(Assert.java:88)
	at org.junit.Assert.failNotEquals(Assert.java:834)
	at org.junit.Assert.assertEquals(Assert.java:645)
	at org.junit.Assert.assertEquals(Assert.java:631)
	at com.hazelcast.scheduledexecutor.ScheduledExecutorServiceBasicTest.scheduleWithRepetition(ScheduledExecutorServiceBasicTest.java:786)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:95)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:56)
	at java.lang.reflect.Method.invoke(Method.java:620)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at com.hazelcast.test.FailOnTimeoutStatement$CallableStatement.call(FailOnTimeoutStatement.java:106)
	at com.hazelcast.test.FailOnTimeoutStatement$CallableStatement.call(FailOnTimeoutStatement.java:98)
	at java.util.concurrent.FutureTask.run(FutureTask.java:274)
	at java.lang.Thread.run(Thread.java:809)
```

Standard output:
```
Hiccups measured while running test 'scheduleWithRepetition(com.hazelcast.scheduledexecutor.ScheduledExecutorServiceBasicTest):'
01:30:25, accumulated pauses: 46 ms, max pause: 1 ms, pauses over 1000 ms: 0
01:30:30, accumulated pauses: 56 ms, max pause: 10 ms, pauses over 1000 ms: 0
01:30:35, accumulated pauses: 74 ms, max pause: 8 ms, pauses over 1000 ms: 0
01:30:40, accumulated pauses: 191 ms, max pause: 10 ms, pauses over 1000 ms: 0
01:30:45, accumulated pauses: 1264 ms, max pause: 1235 ms, pauses over 1000 ms: 1
01:30:50, accumulated pauses: 50 ms, max pause: 8 ms, pauses over 1000 ms: 0
01:30:55, accumulated pauses: 74 ms, max pause: 9 ms, pauses over 1000 ms: 0
01:31:00, accumulated pauses: 30 ms, max pause: 1 ms, pauses over 1000 ms: 0
01:31:05, accumulated pauses: 26 ms, max pause: 0 ms, pauses over 1000 ms: 0
01:31:10, accumulated pauses: 32 ms, max pause: 3 ms, pauses over 1000 ms: 0
01:31:15, accumulated pauses: 42 ms, max pause: 5 ms, pauses over 1000 ms: 0
01:31:20, accumulated pauses: 25 ms, max pause: 0 ms, pauses over 1000 ms: 0
```

--- a/hazelcast/src/test/java/com/hazelcast/scheduledexecutor/ScheduledExecutorServiceBasicTest.java
+++ b/hazelcast/src/test/java/com/hazelcast/scheduledexecutor/ScheduledExecutorServiceBasicTest.java
@@ -780,7 +780,7 @@ public class ScheduledExecutorServiceBasicTest extends ScheduledExecutorServiceT
 
         IScheduledFuture future = s.scheduleAtFixedRate(new ICountdownLatchRunnableTask("latch"), 0, 1, SECONDS);
 
-        latch.await(10, SECONDS);
+        latch.await(20, SECONDS);
         future.cancel(false);
 
         assertEquals(0, latch.getCount());

